packages/contracts/contracts/minter-suite/Minters/MinterSetPriceERC20V5.sol

```diff
@@ -189,16 +184,19 @@ contract MinterSetPriceERC20V5 is
      * @notice Purchases a token from project `projectId`.
      * @param projectId Project ID to mint a token on.
      * @param coreContract Core contract address for the given project.
+     * @param purchasePricePerToken Maximum price of token being allowed by the purchaser, no decimal places
      * @return tokenId Token ID of minted token
      */
     function purchase(
         uint256 projectId,
-        address coreContract
+        address coreContract,
+        uint256 purchasePricePerToken
```

Comment:
nit: super minor nit, but WDYT about calling this `maxPricePerToken` to be a little more idiomatic?
(applies to all purchase & purchaseTo functions across both minters)

Reply:
Good call! Updated